packages/lambda-powertools-middleware-log-timeout/__tests__/index.js

```diff
@@ -0,0 +1,68 @@
+const consoleLog = jest.spyOn(global.console, 'log')
+
+const middy = require('middy')
+const logTimeoutMiddleware = require('../index')
+
+const delay = (millis) => new Promise((resolve) => setTimeout(resolve, millis))
```

Comment:
shouldn't be using delays in tests as they're flakey and slow down the tests (see next comment)